devel/meson/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                meson
 version             1.5.0
-revision            0
+revision            1
```

Comment:
no need to increase the "revision" here as this port only installs symlinks, which have not changed.